pkg/server/server.go

```diff
@@ -3125,7 +3125,7 @@ func (s *BgpServer) ListStatement(ctx context.Context, r *api.ListStatementReque
 	var l []*api.Statement
 	s.mgmtOperation(func() error {
 		s := s.policy.GetStatement(r.Name)
-		l = make([]*api.Statement, len(s))
+		l = make([]*api.Statement, 0)
```

Comment:
l = make([]*api.Statement, 0, len(s))
For consistency with other places and efficiency, can you fix in the above way.

Reply:
sure, updated